site/profile/manifests/core/common.pp

```diff
@@ -41,6 +41,7 @@
   Boolean $manage_ipa = true,
   Boolean $disable_ipv6 = false,
   Boolean $manage_firewall = true,
+  Boolean $install_x2go = true,
```

Comment:
`profile::core::common` is intended to be included by all roles so by setting `true` here, x2go will be installed on all hosts, is that the intent?

Reply:
No, that's not. I will turn it off. Thanks.